contracts/script/DeployLiquity2.s.sol

```diff
@@ -789,23 +812,24 @@ contract DeployLiquity2Script is DeployGovernance, UniPriceConverter, StdCheats,
         // deploy Curve StableswapNG pool
         address[] memory coins = new address[](2);
         coins[BOLD_TOKEN_INDEX] = address(_boldToken);
-        coins[USDC_INDEX] = address(USDC);
+        coins[OTHER_TOKEN_INDEX] = address(_otherToken);
         uint8[] memory assetTypes = new uint8[](2); // 0: standard
         bytes4[] memory methodIds = new bytes4[](2);
         address[] memory oracles = new address[](2);
-        ICurveStableswapNGPool curvePool = curveStableswapFactory.deploy_plain_pool(
-            "USDC-BOLD",
-            "USDCBOLD",
-            coins,
-            200, // A
-            1000000, // fee
-            20000000000, // _offpeg_fee_multiplier
-            865, // _ma_exp_time
-            0, // implementation id
-            assetTypes,
-            methodIds,
-            oracles
-        );
+
+        ICurveStableswapNGPool curvePool = curveStableswapFactory.deploy_plain_pool({
+            name: string.concat("BOLD/", _otherToken.symbol(), " Pool"),
+            symbol: string.concat("BOLD", _otherToken.symbol()),
+            coins: coins,
+            A: 100,
+            fee: 4000000,
+            offpeg_fee_multiplier: 20000000000,
+            ma_exp_time: 866,
+            implementation_id: 0,
+            asset_types: assetTypes,
+            method_ids: methodIds,
+            oracles: oracles
+        });
```

Comment:
Got these parameters by attempting to create a pool on https://curve.fi using the Crypto Collateralized Stablecoins preset, then decoding the hex data. @ColinPlatt does this look good?